src/options.js

```diff
@@ -15,11 +15,11 @@ export const DESKTOP_CANVAS_SIZE = {
 
 export const BALL_RADIUS = 5
 export const COLORS = {
-  death: '#bc4408',
-  recovered: '#0880bc',
-  incubating: '#f5d11b',
-  infected: '#c898a2',
-  well: '#adbc08',
+  death: '#c10000',
+  recovered: '#0031db',
```

Comment:
A ball having the app is identified by a black circle (see `app_installed` color below), when its state turns to `recovered` it becomes hard to be see the black circle.
Consider to update the `app_installed` color as well.